openedx/core/djangoapps/user_authn/views/login.py

```diff
@@ -56,29 +56,19 @@ def _do_third_party_auth(request):
             u"with backend_name {backend_name}".format(
                 username=username, backend_name=backend_name)
         )
-        message = _(
+        message = Text(_(
             u"You've successfully logged into your {provider_name} account, "
-            u"but this account isn't linked with an {platform_name} account yet."
-        ).format(
-            platform_name=platform_name,
-            provider_name=requested_provider.name,
-        )
-        message += "<br/><br/>"
-        message += _(
+            u"but this account isn't linked with an {platform_name} account yet. {br_tags}"
```

Comment:
I know you didn't add this, but do we have a way of handling `an {platform_name}` in other translation strings?  Seems odd to assume the name starts with a vowel.

Reply:
@robrap Looks like no other place in the platform where we are using `an` with platform name. Seems like there is no mechanism used to handle this, I have removed it from here as well.